pkg/auth/providers/local/local_auth_provider.go

```diff
@@ -425,6 +424,18 @@ func indexField(field string, maxindex int) []string {
 	for i := 2; i <= maxindex; i++ {
 		fieldIndexes = append(fieldIndexes, field[0:i])
 	}
+
+	splitToLower := func(s string) []string {
+		var lowers []string
+		for _, v := range strings.Split(s, " ") {
+			lowers = append(lowers, strings.ToLower(v))
+		}
+
+		return lowers
+	}
+
+	fieldIndexes = append(fieldIndexes, splitToLower(field)...)
```

Comment:
Should we be deduping indexes?

Reply:
Good idea, done.

I haven't touched Group indexing in this fix, but a lot of the same things might apply there too.